web/index.html

```diff
@@ -10,181 +11,171 @@
-      <td><a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-cd.iso.md5">md5</a> - 
-          <a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-cd.iso.sha1">sha1</a></td>
-      <td>
-       <a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-cd.iso.torrent">Torrent</a>
-      </td>
-      <td>48 mil páginas, 5% de las imágenes</td>
-     </tr>
-
-     <tr>
-      <td>Tarball, medio</td>
-      <td>2.4 GB</td>
-      <td><a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-tarmed.tar.xz.md5">md5</a> - 
-          <a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-tarmed.tar.xz.sha1">sha1</a></td>
-      <td>
-       <p><a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-tarmed.tar.xz.torrent">Torrent</a></p>
-      </td>
-      <td>280 mil páginas, 8% de las imágenes</td>
-     </tr>
-
-     <tr>
-      <td>DVD</td>
-      <td>4.2 GB</td>
-      <td><a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-dvd5.iso.md5">md5</a> - 
-          <a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-dvd5.iso.sha1">sha1</a></td>
-      <td>
-       <p><a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-dvd5.iso.torrent">Torrent</a></p>
-      </td>
-      <td>800 mil páginas, 8% de las imágenes</td>
-     </tr>
-
-     <tr>
-      <td>Tarball, grande</td>
-      <td>14 GB</td>
-      <td><a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-tarbig.tar.xz.md5">md5</a> - 
-          <a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-tarbig.tar.xz.sha1">sha1</a></td>
-      <td>
-       <p><a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-tarbig.tar.xz.torrent">Torrent</a></p>
-      </td>
-      <td>Un millón de páginas, 85% de las imágenes</td>
-     </tr>
-
-    </tbody>
-   </table>
-
-   <p>Todos los archivos (torrentes, checksums, etc) los podés ver <a href="http://cdpedia.python.org.ar/images/es/current/">acá</a>.</p>
+    <h2 name="download">{{ subtitle_download_it }}</h2>
+
+    <p>{{ download_it_text_a }}</p>
+
+    <p>{{ download_it_text_b }}</p>
+
+    <table class="table table-bordered table-striped table-hover">
+      <thead>
+        <tr>
+          <th>{{ table_head_format }}</th>
+          <th>{{ table_head_size }}</th>
+          <th>{{ table_head_checksums }}</th>
+          <th>{{ table_head_download }}</th>
+          <th>{{ table_head_content }}</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>{{ table_body_cd }}</td>
+          <td>{{ table_body_cd_weight }}</td>
+          <td>
+            <a href="http://cdpedia.python.org.ar/images/es/20170627/cdpedia-es-0.8.4-20170627-cd.iso.md5">{{ table_body_checksum_md5 }}</a> - 
```

Comment:
Esto tendría que salir de algunas variables, iterando las distintas imágenes con un "for", no?